apps/zotonic_mod_search/src/support/search_query.erl

```diff
@@ -300,8 +301,20 @@ qterm({content_group, ContentGroup}, Context) ->
 qterm({id_exclude, Ids}, Context) when is_list(Ids) ->
     %% id_exclude=resource-id
     %% Exclude an id or multiple ids from the result
-    Es = lists:map(fun(Id) -> {id_exclude, Id} end, Ids),
-    qterm(Es, Context);
+    RscIds = lists:filtermap(
+        fun(Id) ->
```

Comment:
This function and at line 331 - look the same, maybe it should be moved to a separate function or macro